cmake/Modules/FindSYCL/run_sycl.cmake

```diff
@@ -129,41 +140,44 @@ if(SYCL_result)
   message(FATAL_ERROR "Error generating file ${generated_file}")
 endif()
 
-# Parse *.d file to retrieve included headers. These headers are dependencies
-# of custom compilation command. Inform cmake to scan these files and
-# retrigger compilation if anything change in these headers.
-SYCL_execute_process(
-  "Generating temporary cmake readable file: ${cmake_dependency_file}.tmp"
-  COMMAND "${CMAKE_COMMAND}"
-  -D "input_file:FILEPATH=${SYCL_generated_dependency_file}"
-  -D "output_file:FILEPATH=${cmake_dependency_file}.tmp"
-  -D "verbose=${verbose}"
-  -P "${SYCL_make2cmake}"
-  )
+# TODO: enable incremental build on Windows
+if(NOT WIN32)
```

Comment:
NOT MSVC? I m not sure whether there is another compiler on Win32, which might support dependencies checking.

Reply:
Thanks, updated.